### DISCUSSION TITLE
After 'npm run build' and 'npm run export' the manually url route doesn't work

### DISCUSSION THREAD

**Discussion by HerbertCJ** [vercel/next.js#47819] [Help]
### Summary

I have an application using React 18.1.0 and Next 12.1.6. It's work nicelly when i'm using on development testing 'npm run dev', all the routes changes when I write manually on URL browser like: www.example.com/account, he goes to account, but when I try to send to me server, or test it locally with 'serve -s out', It doesn't work anymore.
When I try to acess manually URL /account, the page just redirects me to /home, or in some cases the URL remains /account, but the content who is show is the home content, When I export and created the folder Out, just have one index.html, and my folders, like home, account, etc... But this index.html of the root folder of OUT, don't have the routing system like when was in development mode, why?

Is there something I can do to have my 'npm run export' and works equal the development mode, the routes thing...?

Thank you.

**Comment by icyJoseph** [comment]
In Next 12, export should work out of the box, building each page as they exist in the `pages` directory. 

- Could you share how is your application structured? The pages directory structure.

Maybe you are using an unsupported feature?

- https://nextjs.org/docs/advanced-features/static-html-export#unsupported-features

**Reply by HerbertCJ** [reply]
To solve this, I added react-router-dom, so when I export my files, he created a index.html root file, and there have all routes pages, but now, I update the next to the latest version, and not don't work anymore with router-dom, and with nothing, How I can solve it?
Now I'm using Next 13.2.4 and I want when I do npm run build && npm run export, my static files have some routing, like if I enter in /account it goes to /account and not redirect me to /home or /
